src/plugins/expressions/public/loader.test.ts

```diff
@@ -19,10 +19,12 @@
 
 import { first, skip, toArray } from 'rxjs/operators';
 import { loader, ExpressionLoader } from './loader';
-import { ExpressionDataHandler } from './execute';
 import { Observable } from 'rxjs';
 import { ExpressionAstExpression, parseExpression, IInterpreterRenderHandlers } from '../common';
 
+// eslint-disable-next-line
+const { __getLastExecution } = require('./services');
```

Comment:
Why did you have to use require here?

Reply:
Because the required `__getLastExecution` does not exist in the `./services` module and is only created by the Jest mocks in this test suite.